### PR TITLE
navbar refactored

### PR DESCRIPTION
**Pull request by CrhistianT7**

### CODE REVIEW

File: src/components/Header/Header.styles.ts [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+  @media only screen and (min-width: ${breakpoints.lg}) {
+    position: relative;
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-end;
+    gap: 3rem;
+    align-items: center;
+    background: none;
+    top: 0;
+    height: 4%.5rem;
```
Comment (GustavoUT22) [inline src/components/Header/Header.styles.ts:51]: correct that

File: src/components/Header/Header.tsx [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+  HamburgerIcon,
+  HeaderWrapper,
+  MobileNavigation,
+  StyledHeader,
+} from './Header.styles'
+import logo from '../../assets/images/logo/only-text.svg'
+import LanguageSwitcher from 'components/LanguageSwitcher/LanguageSwitcher'
+import NavLink from 'ui/NavLink/NavLink'
+import ButtonLink from 'ui/ButtonLink/ButtonLink'
+import { motion, AnimatePresence } from 'framer-motion'
```
Comment (GustavoUT22) [inline src/components/Header/Header.tsx:18]: change the position to  top

File: src/components/Header/Header.tsx [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+      href: '/about-us',
+    },
+    {
+      name: 'Blog',
+      href: '/blog',
+    },
+  ]
+
+  const handleIsOpen = () => {
+    console.log('here')
```
Comment (GustavoUT22) [inline src/components/Header/Header.tsx:55]: console.log is not necessary

File: src/ui/NavLink/NavLink.styles.ts [hunk 4]
```diff
@@ -9,23 +9,32 @@ export const StyledListItem = styled.li`
 `
 
 export const StyledAnchor = styled(NavLink)`
-  color: ${colors.secondary.main};
+  color: ${colors.primary.main};
   text-decoration: none;
   font-size: 14px;
```
Comment (GustavoUT22) [inline src/ui/NavLink/NavLink.styles.ts:14]: change to rem

File: src/ui/NavLink/NavLink.styles.ts [hunk 5]
```diff
@@ -9,23 +9,32 @@ export const StyledListItem = styled.li`
...
+  }
+
+  &.is-active {
+    color: red;
+  }
 
   &:hover {
-    border-bottom: 2px solid #d20427;
+    color: ${colors.primary.hover};
     border-radius: 2px;
```
Comment (GustavoUT22) [inline src/ui/NavLink/NavLink.styles.ts:30]: rem please

File: src/ui/NavLink/NavLink.styles.ts [hunk 6]
```diff
@@ -9,23 +9,32 @@ export const StyledListItem = styled.li`
...
   }
 
   &:active,
   &:focus {
-    outline: 1px solid #d20427;
+    outline: 2px solid ${colors.primary.focus};
     border-radius: 6px;
```
Comment (GustavoUT22) [inline src/ui/NavLink/NavLink.styles.ts:36]: change to rem

File: src/components/Header/Header.tsx [hunk 7]
```diff
@@ -0,0 +1,158 @@
...
+  StyledHeader,
+} from './Header.styles'
+import logo from '../../assets/images/logo/only-text.svg'
+import LanguageSwitcher from 'components/LanguageSwitcher/LanguageSwitcher'
+import NavLink from 'ui/NavLink/NavLink'
+import ButtonLink from 'ui/ButtonLink/ButtonLink'
+import { motion, AnimatePresence } from 'framer-motion'
+
+const Header: React.FC = () => {
+  const location = useLocation()
```
Comment (GustavoUT22) [inline src/components/Header/Header.tsx:21]: why useLocation?

File: src/components/Header/Header.styles.ts (hunk 1 above)
Comment (CrhistianT7) [reply src/components/Header/Header.styles.ts:51]: done

File: src/components/Header/Header.tsx (hunk 2 above)
Comment (CrhistianT7) [reply src/components/Header/Header.tsx:18]: done

File: src/components/Header/Header.tsx (hunk 3 above)
Comment (CrhistianT7) [reply src/components/Header/Header.tsx:55]: done

File: src/ui/NavLink/NavLink.styles.ts (hunk 4 above)
Comment (CrhistianT7) [reply src/ui/NavLink/NavLink.styles.ts:14]: done

File: src/ui/NavLink/NavLink.styles.ts (hunk 5 above)
Comment (CrhistianT7) [reply src/ui/NavLink/NavLink.styles.ts:30]: done

File: src/ui/NavLink/NavLink.styles.ts (hunk 6 above)
Comment (CrhistianT7) [reply src/ui/NavLink/NavLink.styles.ts:36]: done

File: src/components/Header/Header.tsx (hunk 7 above)
Comment (CrhistianT7) [reply src/components/Header/Header.tsx:21]: it was just a proof of concept, removed

**Review by GustavoUT22 — APPROVED** [review]
Fine !

**Pull request merged**